src/android/StatusBar.java

```diff
@@ -202,25 +202,16 @@ private void setStatusBarTransparent(final boolean transparent) {
     }
 
     private void setStatusBarStyle(final String style) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-            if (style != null && !style.isEmpty()) {
-                Window window = cordova.getActivity().getWindow();
-                View decorView = window.getDecorView();
-                WindowInsetsControllerCompat windowInsetsControllerCompat = WindowCompat.getInsetsController(window, decorView);
-
-                String[] darkContentStyles = { STYLE_DEFAULT };
-                String[] lightContentStyles = { STYLE_LIGHT_CONTENT };
-
-                if (Arrays.asList(darkContentStyles).contains(style.toLowerCase())) {
-                    windowInsetsControllerCompat.setAppearanceLightStatusBars(true);
-                    return;
-                }
-
-                if (Arrays.asList(lightContentStyles).contains(style.toLowerCase())) {
-                    windowInsetsControllerCompat.setAppearanceLightStatusBars(false);
-                    return;
-                }
-
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && !style.isEmpty()) {
```

Comment:
Noting that the `style != null` check has been removed, but it should be fine as the usage of this method isn't directly exposed to the JS webview and in all usages has a guaranteed string value.